src/main/app/src/app/shared/batch-progress/batch-process.service.ts

```diff
@@ -80,14 +78,14 @@ export class BatchProcessService {
   }
 
   showProgress(message: string) {
-    const dialogRef = this.utilService.openProgressDialog({
+    this.utilService.openProgressDialog({
       progressPercentage: this.progress,
       message,
     });
   }
 
   stop() {
-    this.snackBarRef?.dismiss();
+    this.utilService.closeProgressDialog();
```

Comment:
Not part of this PR I know but everywhere I see the word 'util' in source code it's a bit of a red flag as in: bad practise to name code 'util' because it says nothing really. Better to split it up and name it for what it is. :-) For a rainy day maybe.